packages/zoe/src/contractFacet/typeGuards.js

```diff
@@ -0,0 +1,27 @@
+import { M } from '@endo/patterns';
+import { AmountKeywordRecordShape, IssuerRecordShape } from '../typeGuards.js';
+
+export const ZcfSeatShape = M.remotable('zcfSeat');
+
+export const ZcfMintI = M.interface('ZcfMint', {
+  getIssuerRecord: M.call().returns(IssuerRecordShape),
+  mintGains: M.call(AmountKeywordRecordShape)
+    .optional(ZcfSeatShape)
+    .returns(ZcfSeatShape),
+  burnLosses: M.call(AmountKeywordRecordShape, ZcfSeatShape).returns(),
+});
+
+export const ZcfI = M.interface(
+  'ZCF',
+  {
+    makeInvitation: M.call(M.raw(), M.string())
+      .optional(M.record(), M.pattern())
+      .returns(M.promise()),
+    // TODO raw argument should be mandatory, but there's a no-arg call in
+    // the documentation repo
+    setTestJig: M.call().optional(M.raw()).returns(),
```

Comment:
maybe we can fix that before this merges.
https://github.com/Agoric/documentation/blob/89a7dd53cd59b4008a36d23abdfa5665d1852336/snippets/tools/zcfTesterContract.js#L12

Reply:
https://github.com/Agoric/agoric-sdk/pull/9533 takes the opposite tack, correcting the static type to declare this parameter optional. If approved, then the guard in this PR becomes correct and I will remove the `TODO` comment indicating a problem.